analysis/common_variables.py

```diff
@@ -636,6 +640,349 @@ def generate_common_variables(index_date_variable, exposure_end_date_variable, o
+    # ),
+
+    # Combined history of parkinson_disease
+    cov_bin_history_parkinson_disease = patients.minimum_of(
+        "tmp_cov_bin_history_parkinson_disease_snomed", "tmp_cov_bin_history_parkinson_disease_hes", 
+        #"tmp_cov_bin_history_parkinson_disease_death",
+    ),
+
+## History of Multiple Sclerosis
+    # Primary
+    tmp_cov_bin_history_multiple_sclerosis_snomed=patients.with_these_clinical_events(
+        multiple_sclerosis_snomed,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+    # HES
+    tmp_cov_bin_history_multiple_sclerosis_hes=patients.admitted_to_hospital(
+        with_these_diagnoses=multiple_sclerosis_icd10,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+
+    # ONS
+    # tmp_cov_bin_history_multiple_sclerosis_death=patients.with_these_codes_on_death_certificate(
+    #     multiple_sclerosis_icd10,
+    #     returning="binary_flag",
+    #     on_or_before=f"{index_date_variable} - 1 day",
+    #     return_expectations={"incidence": 0.1,
+    #     },
+    # ),
+
+    # Combined history of multiple_sclerosis
+    cov_bin_history_multiple_sclerosis = patients.minimum_of(
+        "tmp_cov_bin_history_multiple_sclerosis_snomed", "tmp_cov_bin_history_multiple_sclerosis_hes", 
+        #"tmp_cov_bin_history_multiple_sclerosis_death",
+    ),
+
+    # History of restless leg syndrome
+    # Primary
+    cov_bin_history_restless_leg_syndrome=patients.with_these_clinical_events(
+        restless_leg_syndrome_snomed,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+
+    # History of REM sleep disorder
+    ## REM sleep disorder (Condition on the clinical pathway to outcome of interest)
+    # Primary
+    tmp_cov_bin_history_rem_sleep_disorder_snomed=patients.with_these_clinical_events(
+        rem_sleep_disorder_snomed,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+    # HES
+    tmp_cov_bin_history_rem_sleep_disorder_hes=patients.admitted_to_hospital(
+        with_these_diagnoses=rem_sleep_disorder_icd10,
+        returning="binary_flag",
+        on_or_before=f"{index_date_variable} - 1 day",
+        return_expectations={"incidence": 0.1,
+        },
+    ),
+    # ONS
+    # tmp_cov_bin_history_rem_sleep_disorder_death=patients.with_these_codes_on_death_certificate(
+    #     rem_sleep_disorder_icd10,
+    #     returning="binary_flag",
+    #     on_or_before=f"{index_date_variable} - 1 day",
+    #     return_expectations={"incidence": 0.1,
+    #     },
+    # ),
+    # Combined rem sleep disorder
+    cov_bin_history_rem_sleep_disorder = patients.minimum_of(
```

Comment:
As per meeting 09/01/2023, remove the variable `cov_bin_history_rem_sleep_disorder`. The temporary variables will be used later for `cov_bin_history_parkison_risk_conditions` - please move this code down.

Reply:
The temporary variables will be used later for cov_bin_history_parkison_risk_conditions - please move this code down so that all this information is together.

Reply:
See review comment instead.